client/src/components/LoginForm.js

```diff
@@ -56,25 +60,37 @@ const LoginForm = () => {
 			</Typography>
 			<FormControl margin="dense">
 				{/* field for email */}
-				<TextField id="login-email" label="Email" type="email" margin="dense" onChange={(e) => {
-					submitObject.email = e.target.value;
-				}} />
+				<TextField
+					id="login-email"
+					label="Email"
+					type="email"
+					margin="dense"
+					autoComplete="off"
+					onChange={(e) => {
+						submitObject.email = e.target.value;
+					}}
+				/>
 
 				{/* Field for password */}
 				<TextField
 					id="login-password"
 					label="Password"
 					type="password"
 					margin="dense"
+					autoComplete="new-password"
 					onChange={(e) => {
 						submitObject.password = e.target.value;
 					}}
 				/>
 
-				<LoginButton variant="contained" size="medium" onClick={() => {
-					setSubmitObjectState(submitObject);
-					tryLogin();
-				}}>
+				<LoginButton
+					variant="contained"
+					size="medium"
+					onClick={() => {
+						setSubmitObjectState(submitObject);
```

Comment:
If you will store the input changes directly in the state, then you will not be required to set this state here